src/ErgoDex/Types.hs

```diff
@@ -0,0 +1,74 @@
+module ErgoDex.Types where
+
+import Prelude (Show, Eq, Integer, ($), (==), (<>), negate)
+
+import           Ledger
+import           Ledger.Value      (AssetClass(..), assetClassValueOf, assetClassValue, Value(..))
+import           PlutusTx.Numeric  (AdditiveSemigroup(..), MultiplicativeSemigroup(..))
+import qualified PlutusTx.AssocMap as Map
+
+import ErgoDex.Contracts.Types
+
+data Lovelace = Lovelace
```

Comment:
you can use https://github.com/input-output-hk/plutus/blob/edd3ce253a9080cd92b785f5a2b48f7d5b09e350/plutus-ledger-api/src/Plutus/V1/Ledger/Ada.hs#L58